docs/alert.md

```diff
@@ -3,6 +3,7 @@
 
 ```jsx
 import { Alert } from 'radiance-ui';
+const SampleContentComponent = () => (<div>Your info is updated!  <a href='http://google.com'> Click here to see changes</a></div>);
```

Comment:
we could just make this into a real example like `<React.Fragment>Click <a href='https://google.com'>here</a> to go to Google!</React.Fragment>`

Reply:
just used div so i don't need to add {" "} whitespace